### ISSUE TITLE
how to cast youtube videos to omxplayer

### ISSUE THREAD

**Issue by ThatsFluke** [popcornmix/omxplayer#725]
how do you put a youtube video onto omxplayer? 

thanks!

**Comment by yxdming** [comment]
Do u mean visit youtube through browser in respberry? Or download the video and play with omxplayer?

**Comment by ThatsFluke** [comment]
@yxdming no, i mean you can stream youtube videos to omxplayer, its just not properly working for me, i get a black screen and audio off the video.

**Comment by ThatsFluke** [comment]
@yxdming i have a feeling its a problem with youtube-dl though.

**Comment by ThatsFluke** [comment]
im making a google chromecast-like operating system, i just need to code the omxplayer to run videos. i have a wireless display protocol and airplay too.

**Comment by bluss** [comment]
what omxplayer supports well is videos where youtube-dl can get you a combined file (video + audio), try using this format setting: `omxplayer $(youtube-dl -f best -g YOUTUBE_VIDEO_URL)`  unfortunately, this basically limits you to 720p video (or worse); the better quality videos use split video / audio streams.
